src/main.rs

```diff
@@ -10,19 +10,18 @@
  * See the License for the specific TON DEV software governing permissions and
  * limitations under the License.
  */
-#[macro_use]
-extern crate clap;
-#[macro_use]
-extern crate serde_json;
-#[macro_use]
-extern crate serde_derive;
+#[macro_use] extern crate clap;
```

Comment:
You may omit this declarations and add macros by `use` in target modules. Or use fully qualified names such as `log::debug!("")`